Parser/Node.cs

```diff
@@ -48,6 +54,36 @@ public record NWriteStmt (bool NewLine, NExpr[] Exprs) : NStmt {
 public record NAssignStmt (Token Name, NExpr Expr) : NStmt {
    public override T Accept<T> (Visitor<T> visitor) => visitor.Visit (this);
 }
+
+// A read statement
+public record NReadStmt (Token[] Tokens) : NStmt {
+   public override T Accept<T> (Visitor<T> visitor) => visitor.Visit (this);
+}
+
+// A call statement 
+public record NCallStmt (Token Name, NExpr[] Exprs) : NStmt {
+   public override T Accept<T> (Visitor<T> visitor) => visitor.Visit (this);
+}
+
+// A if statement 
+public record NIfStmt (NExpr Expr, NStmt[] Stmts) : NStmt {
+   public override T Accept<T> (Visitor<T> visitor) => visitor.Visit (this);
+}

```

Comment:
Convert to:
`public record NIfStmt (NExpr Condition, NStmt IfPart, NStmt ElsePart)`